lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -960,5 +1072,206 @@ define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpe
                     });
                 });
             });
+
+            describe("Discussion Topics", function() {
+                var createCourseWideView, createInlineView,
+                    inlineView, courseWideView, assertCohortedTopics;
+
+                createCourseWideView = function(that) {
+                    createCohortsView(that);
+
+                    courseWideView = new CohortCourseWideDiscussionsView({
+                        el: cohortsView.$('.cohort-discussions-nav').removeClass('is-hidden'),
+                        model: cohortsView.context.discussionTopicsModel,
+                        cohortSettings: cohortsView.cohortSettings
+                    });
+                    courseWideView.render();
+                };
+
+                createInlineView = function(that) {
+                    createCohortsView(that);
+
+                    inlineView = new CohortInlineDiscussionsView({
+                        el: cohortsView.$('.cohort-discussions-nav').removeClass('is-hidden'),
+                        model: cohortsView.context.discussionTopicsModel,
+                        cohortSettings: cohortsView.cohortSettings
+                    });
+                    inlineView.render();
+                };
+
+                assertCohortedTopics = function(view, type) {
+                    expect(view.$('.check-discussion-subcategory-' + type).length).toBe(2);
+                    expect(view.$('.check-discussion-subcategory-' + type + ':checked').length).toBe(1);
+                };
+
+                it("view renders properly", function() {
+                    showAndAssertDiscussionTopics(this);
+                });
+
+                describe("Course Wide", function() {
+
+                    it("save button is disabled initially", function() {
```

Comment:
@muzaffaryousaf Good question. Here's the JS guidelines but it is out-of-date and doesn't describe single quotes. In general, following the style of the file you are in makes sense, so if all the other tests are using double quotes then it is reasonable to do the same. 

https://github.com/edx/edx-platform/wiki/Javascript-standards-for-the-edx-platform

I've added links to all our coding guidelines on the T&L code review guidelines wiki:

https://openedx.atlassian.net/wiki/display/TNL/Code+Review+Guidelines+and+Gotchas